tests/target-aarch64.cpp

```diff
@@ -0,0 +1,8 @@
+// verify that target to aarch64 works
+// REQUIRES: clang
+// RUN: %clang -c -target aarch64-linux-gnu %s
+// RUN: file %s.o|grep -i -q "aarch64"
```

Comment:
`llvm-objdump` is probably better than file here too ;-)
On a more general note, do we really want to check support for a specific backend? What is the motivation and what makes aarch64 different from other targets?

Reply:
To start from somewhere ;)

Reply:
About llvm-objdump, I will update once https://github.com/opencollab/llvm-toolchain-integration-test-suite/pull/51 landed